packages/sources/css/src/components/button/src/index.css

```diff
@@ -198,7 +198,64 @@
 }
 
 /* Button icons  */
-.vtmn-btn-icon {
+.vtmn-btn_icon--alone {
+  width: rem(48px);
+  height: rem(48px);
+  padding: rem(14px);
+}
+
+.vtmn-btn_icon--alone span {
+  font-size: var(--vtmn-typo_text-2-font-size);
+}
+
+.vtmn-btn_icon--along span {
+  padding-right: rem(8px);
+  vertical-align: text-top;
+}
+
+.vtmn-btn_size--small.vtmn-btn_icon--alone {
+  width: rem(32px);
+  height: rem(32px);
+  padding: rem(8px);
+}
+
+.vtmn-btn_size--small.vtmn-btn_icon--alone span {
+  font-size: var(--vtmn-typo_text-3-font-size);
+}
+
+.vtmn-btn_size--small.vtmn-btn_icon--along span {
+  padding-right: rem(6px);
+}
+
+.vtmn-btn_size--medium.vtmn-btn_icon--alone {
+  width: rem(48px);
+  height: rem(48px);
+  padding: rem(14px);
+}
+
+.vtmn-btn_size--medium.vtmn-btn_icon--alone span {
+  font-size: var(--vtmn-typo_text-2-font-size);
+}
+
+.vtmn-btn_size--medium.vtmn-btn_icon--along span {
+  padding-right: rem(8px);
+}
+
+.vtmn-btn_size--large.vtmn-btn_icon--alone {
+  width: rem(64px);
+  height: rem(64px);
+  padding: rem(20px);
+}
+
+.vtmn-btn_size--large.vtmn-btn_icon--alone span {
+  font-size: var(--vtmn-typo_title-5-font-size);
+}
+
+.vtmn-btn_size--large.vtmn-btn_icon--along span {
+  padding-right: rem(12px);
+}
+
+/* .vtmn-btn-icon {
```

Comment:
Why in this PR?

Reply:
I deleted all commits on button component